connector-catalog/src/main/java/io/syndesis/connector/catalog/ConnectorCatalog.java

```diff
@@ -51,7 +67,124 @@ public ConnectorCatalog(ConnectorCatalogProperties props) {
         }
     }
 
+    private void prefetchConnectors() {
+        try (InputStream is = getClass().getResourceAsStream(CONNECTOR_SCHEMAS)) {
+            if (is != null) {
+                ArrayNode descriptors = (ArrayNode) new ObjectMapper().readTree(is);
+                for (JsonNode descriptor : descriptors) {
+                    JsonNode gav = descriptor.get("gav");
+                    if (gav == null) {
```

Comment:
don't we control what's in `/META-INF/camel/camel-meta.json` when would `gav` be `null`?

Reply:
Yes, we do. So maybe better throw an exception then except ignoring it?